assets/src/scss/layout/_footer.scss

```diff
@@ -61,7 +61,9 @@
   }
 
   li {
-    font-size: $font-size-sm;
+    _-- {
```

Comment:
This variable is overridden from [here](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/base/_typography.scss#L11-L19) and we could potentially use `var(--text--x-large-and-up--font-size`. However, the new identity has 3 different typos with different `font-sizes`, so `var(--text--x-large-and-up--font-size` won't be an standard value. That's why I preferred to use an specific footer's CSS Variable.